ape_alchemy/providers.py

```diff
@@ -32,6 +32,10 @@ class AlchemyFeatureNotAvailable(AlchemyProviderError):
 
 
 class MissingProjectKeyError(AlchemyProviderError):
```

Comment:
We could consider moving this to an `exceptions.py` file to be consistent.

Reply:
would I move all 3 errors to the `exceptions.py` file?

Reply:
Yes, if we make a `exceptions.py`, let's put all of them in there